easycv/models/detection/detectors/yolox_edge/yolox_edge.py

```diff
@@ -1,7 +1,8 @@
 # Copyright (c) Alibaba, Inc. and its affiliates.
```

Comment:
1. This refactor should be done with yolox-edge author，if anyone PR should be backward compatibility for all code， yolox_edge should be refactor by those who PR detection/detectors/detection.py rather than in this YOLOX-PAI PR.
2.  A github PR code review should not using commanding tone so often